tfhe/docs/fundamentals/encrypted-prf.md

```diff
@@ -1,25 +1,51 @@
 # Generate encrypted pseudo random values
 
-This document gives an example of generating pseudo random values in FHE that are not known by the server.
+This document explains the mechanism and steps to generate an oblivious encrypted random value using only server keys.
+
+The goal is to give to the server the possibility to generate a random value, which will be obtained in a encrypted format and will remain unknown to the server. The implementation is based on [this article](https://eprint.iacr.org/2024/665).
+
+In *TFHE-rs*, this is possible through the method `generate_oblivious_pseudo_random` of `FheUint` and `FheInt`.
+It relies on the use of the usual server keys.
+The method takes two inputs: a seed `Seed`, which could be any `u128` value and an upper bound on the range of the random values `random_bits_count`. 
+It outputs a `FheIntN` or `FheUintN`, where `N` is the number of bits of the homomorphic integer. 
```

Comment:
this is not accurate anymore, the bounded method takes the random_bit_count

Reply:
Fixed